flashcard.rb

```diff
@@ -0,0 +1,12 @@
+class FlashCard
+  attr_reader :term , :definition
+
+  def initialize(args = {})
+    @correct = false
```

Comment:
Without a reader, a writer or any other methods affecting this attribute, how will you ever change it?